modules/k6/src/main/java/org/testcontainers/k6/K6Container.java

```diff
@@ -0,0 +1,89 @@
+package org.testcontainers.k6;
+
+import org.apache.commons.io.FilenameUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class K6Container extends GenericContainer<K6Container> {
```

Comment:
I just realized there is no wait strategy defined and neither exposed ports. But, I just found that there is a [REST API](https://k6.io/blog/how-to-control-a-live-k6-test/), do you think it will help to add 6565 as exposed ports and getEndpoint that returns the `"http://" + getHost() + ":" + getMappedPort(6565)`?

Reply:
True...I can add that in.

Reply:
let's also add a test, please :)

Reply:
My sincere apologies if I'm not describing this very well 🙇.

I tried this, but it failed with related to availability of the port. The image itself does not even expose the REST port ([Dockerfile](https://github.com/grafana/k6/blob/master/Dockerfile)).

Use of the REST API is a pretty specific scenario for external control and would be a very different use case, e.g. changing options within the script you just passed in. The intent is for long-running load tests. 

The workflow for this is to have the passed-in test script execute then immediately exit the container. It's not like a database remaining alive for interaction from the test case.